pom.xml

```diff
@@ -815,8 +815,8 @@
           <artifactId>maven-compiler-plugin</artifactId>
           <version>${maven.compiler.plugin.version}</version>
           <configuration>
-            <source>1.7</source>
-            <target>1.7</target>
+            <source>1.8</source>
```

Comment:
Is there a specific reason for the version upgrade?

Reply:
I don't recollect the reason now. I reverted to 1.7